examples/cra/package.json

```diff
@@ -16,7 +16,7 @@
     "react-dom": "^18.0.0",
     "react-scripts": "5.0.1",
     "typescript": "^4.9.5",
-    "wagmi": "^0.12.0",
+    "wagmi": "^0.12.5",
```

Comment:
The wagmi package I needed was 0.12.4 I think (it has the `isDawn` flags) and `^0.12.0` for some reason wasn't installing the latest package version. So manually forced it to have the next minimum version I needed